projects/optic/src/__tests__/integration/workspaces/ruleset-publish/valid-js-file/rules.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  name: 'asdasdasdasdasdasd - the best ruleset', // this name is so that this is compressable
```

Comment:
@nalanj Ok this is fun, the brotli library we use sometimes returns `null` if the data we pass it cannot be compressed. The reason we haven't run into this before is because the data we try compress (in CLIDiff) is _always_ compress-able. Likely rulesets will also always be compress-able as well, but not guaranteed.

I'm going to address this in a follow up PR

Reply:
Where's the info about it sometimes returning null?

Reply:
See https://github.com/foliojs/brotli.js/issues/36. Feels like that might be what you're hitting.